docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -18,19 +18,40 @@ Install a nested OCP cluster running on VMs within a management OCP cluster
 Before creating a guest cluster, the Hypershift operator and hypershift cli tool
 must be installed.
 
+### Build the HyperShift CLI
+
 The command below builds latest hypershift cli tool from source and places the
 cli tool within the /usr/local/bin directory.
 
-```shell
-podman run --rm --privileged -it -v \
-$PWD:/output docker.io/library/golang:1.18 /bin/bash -c \
-'git clone https://github.com/openshift/hypershift.git && \
-cd hypershift/ && \
-make hypershift && \
-mv bin/hypershift /output/hypershift'
-
-sudo mv $PWD/hypershift /usr/local/bin
-```
+=== "Podman"
+  
+    ```shell
+    podman run --rm --privileged -it -v \
+    $PWD:/output docker.io/library/golang:1.18 /bin/bash -c \
+    'git clone https://github.com/openshift/hypershift.git && \
+    cd hypershift/ && \
+    make hypershift && \
+    mv bin/hypershift /output/hypershift'
+
+    sudo install -m 0755 -o root -g root $PWD/hypershift /usr/local/bin/hypershift
+    rm $PWD/hypershift
+    ```
+
+=== "Docker"
+  
+    ```shell
+    docker run --rm --privileged -it -v \
+    $PWD:/output docker.io/library/golang:1.18 /bin/bash -c \
+    'git clone https://github.com/openshift/hypershift.git && \
+    cd hypershift/ && \
+    make hypershift && \
+    mv bin/hypershift /output/hypershift'
+  
+    sudo install -m 0755 -o root -g root $PWD/hypershift /usr/local/bin/hypershift
+    rm $PWD/hypershift
+    ```
```

Comment:
the podman and docker command are identical except for the use of `podman` or `docker` as the binary, right? If so I'd suggest documenting this just with podman, and saying the command is the same for docker.

Reply:
Added note.